requirements.txt

```diff
@@ -9,4 +9,5 @@ tensorflow>=1.3.0
 mpld3>=0.3
 tqdm>=4.14.0
 pillow>=3.4.2
-imageio>=2.2.0
\ No newline at end of file
+imageio>=2.2.0
+jupyter>=1.0.0
```

Comment:
My opinion is that Jupyter shouldn't be required for the `AxonDeepSeg` package as it isn't required for segmentation but only for the notebook examples (@alzaia is this correct ?).

It would be better to add as an extra requirement (`extras_require`), so people can install by request. Another alternative is to document that they need Jupyter to open notebooks.

Reply:
That's fine – in that case, I feel like the getting started [Getting Started](https://neuropoly.github.io/axondeepseg/documentation.html#getting-started) instructions should be changed. The very first thing you suggest to do after installation is to explore your Jupyter notebooks, and as a user I expected being able to do it out-of-the-box, locally.

I had no instructions to tell me that the Jupyter notebooks are only supposed to be used online (which don't require my installation at all, that I had just verified to pass the tests).

Reply:
Yes @perone you are right. Pretty much everything on ADS can be done without the Jupyter notebooks, but these notebooks are definitely useful for both dev/users. I agree with @mathieuboudreau that the first line in the Getting Started section of the doc is confusing, especially considering that the [getting_started notebook](https://github.com/neuropoly/axondeepseg/blob/master/notebooks/getting_started.ipynb) was mostly made for the Binder link so that users can experiment with an example without having to install the package. The important Jupyter notebooks are listed in the [Jupyter Notebooks](https://neuropoly.github.io/axondeepseg/documentation.html#jupyter-notebooks). I will update the doc in a future PR.

Reply:
@alzaia sound good! For the remaining minor fixes from this branch, do you prefer that I remove the jupyter requirements.txt line and continue with this PR, or would you prefer to cherry-pick them into another future PR?

Reply:
Yes I think we can remove the Jupyter requirement here and you can continue in this PR (just update the title of the PR).

Reply:
@alzaia can you please open an issue and reference the issue in this thread so we don't forget. Thanks!